CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 ### Added
 
 - Add an acceptance test suite to cover a `test cache warm` command on a micro-feature architecture kind of application that is fully statically linked [#1594](https://github.com/tuist/tuist/pull/1594) by [@RomainBoulay](https://github.com/RomainBoulay)
+- Add launch proeprty on the Target level, allowing configuration of Launch Arguments on the target level, just like the Environment property.
```

Comment:
Nitpick: Add support for setting launch arguments at the target level.
Also don't forget to add the URL to the PR and your GitHub user.